scripts/sentry-xcode.sh

```diff
@@ -22,7 +22,7 @@ if [ "$SENTRY_DISABLE_AUTO_UPLOAD" != true ]; then
   /bin/sh -c "$REACT_NATIVE_XCODE_WITH_SENTRY"
 else
   echo "SENTRY_DISABLE_AUTO_UPLOAD=true, skipping sourcemaps upload"
-  /bin/sh -c "$REACT_NATIVE_XCODE"
+  /bin/sh -c "$NODE_BINARY $REACT_NATIVE_XCODE_WITH_SENTRY"
```

Comment:
H: This else branch is when Sentry is disabled, we should execute the scripts as RN does. Also it's a shell script, so node would failed.

```suggestion
/bin/sh -c "$REACT_NATIVE_XCODE"
```